lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
+      self.vcap_data['application_uris'].min_by(&:length)
+    else
+      nil
```

Comment:
> I looked around and we do not appear to be returning `nil` from `else` in other parts of this app

Just searched the code with the regex `^\s*nil$`, and I'm seeing seven places, at least :wink: 

> what would be an example of a method where the return value is not important?

Some examples:

* Most ActiveRecord class methods (`has_many`, `scope`, etc.)
* Controller actions
* [`Ncr::WorkOrder#setup_approvals_and_observers`](https://github.com/18F/C2/blob/d68ca0ed741dcde13a8b66b034493f7b50a9309f/app/models/ncr/work_order.rb#L107-L118)